index.html

```diff
@@ -0,0 +1,58 @@
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css" integrity="sha512-u2Qo5c5wL2fZ5m0z/rqZjLmZ9+vHjE/AqY0rU/QdugnfeE0k8zO6rU2qzUITjtWc6i1Kc2nh/gPmKv5fK/xJwg==" crossorigin="anonymous" >
+
+  </head>
+  <body>
+    <div class="portfolio">
+        <header class="details">
+            <div class="name">
+                <span class="inside-name">Kelvin</span>
+            </div>
+            <nav>
+            <ul class="nav-list">
+                   <li><a href="#" class="item-home">Portfolio</a></li>
+                    <li><a href="#" class="item-about">About</a></li>
+                    <li><a href ="#"class="item-contact">Contact</a></li>
+                    <li><a href="#"class="item-email">
+                        <img src="images/Disabled.png"alt="Disabled">
+                    </a></li>
+                   
+                </ul>
+                </nav>
+               
+        </header>
+        <section class="section-container">
+        <div class="socials-container">
+            <a href="#" class="github">
+                <img src="images/github.png" alt="github">
+            </a>
+            <a href="#" class="linkedin">
+                <img src="images/Linkedin icon.png" alt="linkedin">
+            </a>
+            <a href="#" class="angel-list">
+                <img src="images/angel list.png" alt="angel-list">
+            </a>
+            <a href="#" class="twitter">
+                <img src="images/twitter.png" alt="Twitter">
+            </a>
+            <a href="#" class="medium">
+                <img src="images/medium.png" alt="Medium">
+            </a>
+        </div>
+<section class="about-container">
+    <h2 class="greeting">Hey There.<br> I'm Kelvin</h2>
+    <h1 class="Introduction">I am a Software Developer</h1>
+    <p class="paragraph">I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don't hesitate to contact me.</p>
+</section>
+<div class="vector1">
+<img src="images/Illustration-header.png" alt="vector" >
```

Comment:
- [ ] I would suggest removing this image tag and adding the illustration using CSS in this section. 😸